scripts/compare_crunchers.py

```diff
@@ -28,7 +28,7 @@ def main():
     crunchers_list = [
         #  dc.DatabaseCruncherLeadGas,
         # dc.DatabaseCruncherTimeDepRatio,
-        dc.DatabaseCruncherQuantileRollingWindows,
+        dc.QuantileRollingWindows,
```

Comment:
Why the change to only one class?

Reply:
I'll do the rest in the next release but since I have to re-run everything using this cruncher here I thought I'd split the problem up

Reply:
Also works, can you make a quick issue to remind us to do the rest?